### PR TITLE
Bankist app 

### PR DESCRIPTION
**Pull request by Ivan-Andr**
First coding challenge of section 11 (arrays)

### CODE REVIEW

File: Arrays/script.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  //const dogsJuliaKate = dogsJuliaNew.concat(dogsKate);
+  const dogsJuliaKate = dogsJulia.slice(1, -2).concat(dogsKate);
+
+  //Jonas solution with splice method
+  //const dogsJuliaNew = dogsJulia.slice();
+  //dogsJuliaNew.splice(0, 1);
+  //dogsJuliaNew.splice(-2);
+  //const dogsJuliaKate = dogsJuliaNew.concat(dogsKate);
+
+  dogsJuliaKate.forEach(function (mov, i, arr) {
```
Comment (levnikan) [inline Arrays/script.js:117]: not sure if you've covered arrow functions already, but it easier for me to use ()=> {} instead. E.g. myArray.forEach((element, index, array)=>{})

**Review by levnikan — APPROVED** [review]

**Comment by levnikan** [comment]
:+1:

**Comment by Ivan-Andr** [comment]
Finish coding challenge#2 and add some infornation to git-commands.md

**Review by levnikan — APPROVED** [review]

File: Arrays/script.js [hunk 2]
```diff
@@ -146,3 +146,43 @@ const checkDogs = function (dogsJulia, dogsKate) {
...
+4. Run the function for both test datasets
+
+TEST DATA 1: [5, 2, 4, 1, 15, 8, 3]
+TEST DATA 2: [16, 6, 10, 5, 6, 1, 4]
+
+GOOD LUCK 😀
+*/
+
+const calcAverageHumanAge = function (ages) {
+  const calcHumanAge = ages.map((dogAge, i) => {
```
Comment (levnikan) [inline Arrays/script.js:170]: `i` is not used here, you can remove it

File: Arrays/script.js [hunk 3]
```diff
@@ -146,3 +146,43 @@ const checkDogs = function (dogsJulia, dogsKate) {
...
+    if (dogAge <= 2) {
+      return 2 * dogAge;
+    } else {
+      return 16 + dogAge * 4;
+    }
+  });
+  const filterYongDogs = calcHumanAge.filter((age, i) => age >= 18);
```
Comment (levnikan) [inline Arrays/script.js:177]: same here

File: Arrays/script.js [hunk 4]
```diff
@@ -146,3 +146,43 @@ const checkDogs = function (dogsJulia, dogsKate) {
...
+  const calcAverageAge = filterYongDogs.reduce(
```
Comment (levnikan) [inline Arrays/script.js:178]: here as well, better to call it `averageAge`, because this is not a function that calculates something

**Review by levnikan — APPROVED** [review]

**Pull request merged**